pennylane_sphinx_theme/footer.py

```diff
@@ -4,11 +4,12 @@
 
 FOOTER = {
     "footer_about": {
-        "title": "Xanadu",
+        "title": "PennyLane",
         "description": """\
-        Located in the heart of downtown Toronto, we've brought together
-        exceptional minds from around the world to build quantum computers
-        that are useful and available to people everywhere.
+        PennyLane is an open-source software framework for quantum
+        machine learning, quantum chemistry and quantum computing 
+        with the ability to run on all hardware.
+        Maintained with ♥ by Xanadu.

```

Comment:
Just curious: would this show up as a ♥ or a ❤️?  The first is unicode and the second is an emoji.

Reply:
I guess the former is technically a "Heart Suit" while the latter is a "Red Heart" so, for the sake of screen readers, perhaps it would be better to go with the second one? It probably doesn't matter too much either way.

Reply:
```suggestion
        Maintained with ❤️ by Xanadu.
```